.github/workflows/visual-snapshots-and-acceptance-py2.7.yml

```diff
@@ -58,7 +62,9 @@ jobs:
 
   acceptance:
     # TODO(joshuarli): Convert to py3 with snapshots. See other TODO as well.
+    name: python2.7 acceptance
     runs-on: ubuntu-16.04
+    timeout-minutes: 20
```

Comment:
This might be a bit low, we have a test that may sometimes hit 20 minutes, but generally is < 20.